server/src/main/java/org/elasticsearch/search/dfs/ProfilingDiversifyingChildrenByteKnnVectorQuery.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.search.dfs;
```

Comment:
I think all these queries belong in `org.elasticsearch.search.vectors` I think they will find use outside of `dfs` and strictly speaking they have nothing to do with dfs, just that they are vector queries providing a visited count that is USED in dfs.